specification/compute/resource-manager/readme.python.md

```diff
@@ -52,167 +62,270 @@ batch:
   - tag: package-compute-2015-06
 ```
 
+```yaml $(multiapi) && $(track2)
+clear-output-folder: true
+package-top-folder: compute_preview
+batch:
+  - tag: package-2019-12-01-only
+  - tag: package-2019-11-01-only
+  - tag: package-2019-07-01-only
+  - tag: package-2019-04-01-only
+  - tag: package-2019-03-01-only
+  - tag: package-2018-10-01-only
+  - tag: package-2018-09-30-only
+  - tag: package-compute-only-2018-06
+  - tag: package-compute-2018-04
+  - tag: package-compute-only-2017-12
+  - tag: package-skus-2017-09
+  - tag: package-compute-2017-03
+  - tag: package-compute-2016-04-preview
+  - tag: package-compute-2016-03
+  - tag: package-compute-2015-06
+  - multiapiscript: true
+```
+
+``` yaml $(multiapiscript)
+output-folder: $(python-sdks-folder)/$(package-top-folder)/azure-mgmt-compute/azure/mgmt/compute/
+clear-output-folder: false
+perform-load: false
+```
+
 ### Tag: package-2019-12-01-only and python
 
 These settings apply only when `--tag=package-2019-12-01-only --python` is specified on the command line.
 Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
 
-``` yaml $(tag) == 'package-2019-12-01-only' && $(python)
+``` yaml $(tag) == 'package-2019-12-01-only' && $(python) && !$(track2)
```

Comment:
i think these changes won't be necessary any longer, as the new sdk will be in the same folder